package/yast2-bootloader.changes

```diff
@@ -1,4 +1,86 @@
 -------------------------------------------------------------------
+Wed Aug  2 16:00:44 CEST 2017 - snwint@suse.de
+
+- not everything is a disk, use libstorage's partitionable instead
+
```

Comment:
I'm missing the `3.3.1` version change here... and I can already hear some bot complaining about the lack of fate entry. I'm using `fate#318196` in other packages. Not sure if the bot will enforce us to add it to every single entry or just to the last one